stable/spring-cloud-data-flow/README.md

```diff
@@ -30,7 +30,26 @@ If you are using a cluster that does not have a load balancer (like Minikube) th
 $ helm install --name my-release --set server.service.type=NodePort stable/spring-cloud-data-flow
 ```
 
-RabbitMQ is the default messaging layer, Kafka can be enabled instead by using the following `set` flags to the `helm` command, for example:
+### Data Store
+
+By default, MySQL is deployed with this chart. However, if you wish to use an external database, please use the following `set` flags to the `helm` command to disable MySQL deployment, for example:
+
+`--set mysql.enabled=false`
+
+In addition, you are required to set all fields listed in [External Database Configuration](#external-database-configuration-mysqlenabledfalse).
```

Comment:
`(#external-database-configuration-mysqlenabledfalse)` should be updated when the comment below about `mysql.enabled=false` is removed from the header